src/components/organisms/HotKeysHandler/HotKeysHandler.tsx

```diff
@@ -99,10 +82,22 @@ const HotKeysHandler = () => {
     [rootFilePath]
   );
 
-  useHotkeys(hotkeys.TOGGLE_SETTINGS.key, () => {
-    if (!isInQuickClusterMode) {
-      dispatch(setLeftMenuSelection('settings'));
+  useHotkeys(
+    hotkeys.TOGGLE_SETTINGS.key,
+    () => {
+      if (!isInQuickClusterMode) {
```

Comment:
I suggest adding a notification or a toast so that you cannot toggle settings while you are in cluster mode